cmd/influxd/inspect/build_tsi/build_tsi_test.go

```diff
@@ -345,6 +345,14 @@ func runCommand(t *testing.T, params cmdParams, outs cmdOuts) {
 		isIndex, err := tsi1.IsIndexDir(filepath.Join(params.dataPath, "12345", "autogen", "1", "index"))
 		require.NoError(t, err)
 		require.True(t, isIndex)
+
+		// Check manifest files, at least one index file should be listed in each
+		for i := 0; i < 8; i++ {
+			currentPartition := strconv.Itoa(i)
+			manifest, _, err := tsi1.ReadManifestFile(filepath.Join(params.dataPath, "12345", "autogen", "1", "index", currentPartition, "MANIFEST"))
+			require.NoError(t, err)
+			require.NotNil(t, len(manifest.Files))
```

Comment:
I don't think `len` will ever output `nil`. You could use `require.NotZero` instead.